docs/install_guides/2022.1.1/installing-openvino-from-archive-windows.md

```diff
@@ -0,0 +1,169 @@
+# Install OpenVINO™ Runtime on Windows 10 from an Archive File {#openvino_docs_install_guides_installing_openvino_from_archive_windows}
+
+With the OpenVINO™ 2022.1.1 release, you can download and use archive files to install OpenVINO Runtime.
+
+You can also check the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes) for more information on updates in this release.
+
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter can be installed via [pypi.org](https://pypi.org/project/openvino-dev/) only.
+
+## System Requirements
+
+@sphinxdirective
+.. tab:: Operating Systems
+
+  Microsoft Windows 10, 64-bit
```

Comment:
@ryanloney why don't we support Windows 11? as I know, Windows 10 is not able to properly handle ADL with hybrid cores, so we verify it on Windows 11. Should we add official mentioning of Windows 11? Or perform some extra validation first maybe

Reply:
BTW, it's valid for 2022.1.0 as well